pages/BaseNext/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { RouteComponentProps } from 'react-router-native';
+import Button from '../../components/Base/ButtonItem';
+
+const BaseNext: React.FunctionComponent<RouteComponentProps> = ({
+    history
+}: RouteComponentProps) => {
+      
+    const goToBaseHome = () => {
+        history.push('/')
+    }  
+
+    return (
+        <>
+        <Button
+          label="Back"
+          onButtonPress={goToBaseHome}
+        />
+        </>
```

Comment:
You don't need a fragment here. Please remove fragment

Reply:
I have removed this